### ISSUE TITLE
No gridlines on any plots

### ISSUE THREAD

**Issue by rabutler-usbr** [BoulderCodeHub/Process-CRSS-Res#77]
Something changed and there are no gridlines in any of the plots, and they no longer have the grey background.

@jshirey Can you let me know what may have changed based on your recent work?

**Comment by jshirey** [comment]
Are you talking like the standard figures/plots?  I don't believe I edited
those at all - I kept my cloud plot changes to the cloud figure fxn.
Thinking back on all the steps I ran an old version of the code in January
(which had the cloud figs fxn in it) and those plots all came out fine.  I
did some revisions to the cloud fig plot and then downloaded the most
recent code and threw the function in the plottingFunctions.R, added data
files, and then added in a few lines in Main.R to call the fxn/make sure
the appropriate scenario labels were used in the figures.  I never did run
the full code afterwards though.  Do you want me to look into this more?

Jessica Khaya
Bureau of Reclamation, Lower Colorado Region
Civil Engineer, Hydrologic
Duty Station: CADSWES
421-UCB
Boulder, CO 80309
(303) 492 - 0892

**Comment by rabutler-usbr** [comment]
Yes please - when I run the new code, it changes all of the figures. Maybe it has to do with cowplot?

Not quite sure what's going on.
